### ISSUE TITLE
Text conversion tlg0019

### ISSUE THREAD

**Issue by TDBuck** [PerseusDL/canonical-greekLit#111]
text conversion tlg0019

**Comment by TDBuck** [comment]
Beginning work on tlg0019.tlg009 (the Frogs)

**Comment by lcerrato** [comment]
We (I?) need to ask permission for the English: https://www.une.edu.au/staff-profiles/humanities/mdillon

Was hoping that we could work with the library on permissions acquisition in a formal, organized sense but either myself or @AlisonBabeu can work on reaching out to him.

**Comment by TDBuck** [comment]
Let me know if there is anything I can do to help, or what we want to do
moving forward.

Tim

**Comment by lcerrato** [comment]
Thanks. I've made a note in our tracking spreadsheet about the follow up. I think that converting the protected file (and leaving it where it is until we have more information) is the best way forward for now. Let me know if you need access to that. 
We can also temporarily move this over with restrictions noted in the header with `<availability>` that would permit some Nemo review.

**Comment by TDBuck** [comment]
I have access to the private repo, i can take a crack at it, but probably
after I work my way through the greek.

Should I create a working branch in the protected repo? or a separate
tracking issue? Trying to figure out a best practices.

Tim

**Comment by lcerrato** [comment]
Yes, an issue in the protected repo and you can cross reference it here. You may want to limit this issue to tlg0019.tlg009 in case others want to claim more Aristophanes.

**Comment by TDBuck** [comment]
I'll open the issue when I get to working with the english

**Comment by lcerrato** [comment]
Hi @TDBuck 
where are you with the Greek? 
Can you post an update for @gregorycrane?

I will help finish anything you are too busy to work on.

**Comment by TDBuck** [comment]
@lcerrato I've worked my way through the Frogs, and just need to fill in the missing line numbers. the whole thing is done with </l> tags containing </said> tags to deal with the split lines. I'm not sure how well that will render in Nemo, last time I tried to load it in, it didn't really work.

**Comment by lcerrato** [comment]
Yes, we have two ways of handling the drama which are both inadequate. The "said" option is better in some respects but is bloated and doesn't display. The "sp" option is better for readability but not as elegant for the line breaks.

**Comment by lcerrato** [comment]
@TDBuck Please commit to your branch so that I can take a look at what you have and get it ready for the students, if possible. Thanks. It doesn't matter what state it is in.

**Comment by lcerrato** [comment]
Thanks, @TDBuck! Let's keep this open for now.
